ipalib/constants.py

```diff
@@ -343,3 +343,9 @@
 # Apache's mod_ssl SSLVerifyDepth value (Maximum depth of CA
 # Certificates in Client Certificate verification)
 MOD_SSL_VERIFY_DEPTH = '5'
+
+TRACKING_REQS = {
```

Comment:
Could you please add a comment here stating where this was duplicated from and why so that no refactoring happens down the road?